templates/openstack/architecture.html

```diff
@@ -0,0 +1,263 @@
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <h2>OpenStack under the hood</h2>
+      </div>
+      <div class="col">
+        <div class="p-section--shallow">
+          <div class="p-image-container is-highlighted">
+            {{ image(url="https://assets.ubuntu.com/v1/144affdb-Canonical OpenStack architecture diagram final@2x.png",
+                        alt="",
+                        width="1954",
+                        height="720",
+                        hi_def=True,
+                        attrs={"class": "p-image-container__image"},
+                        loading="lazy") | safe
+            }}
+          </div>
+        </div>
+        <div class="p-section--shallow">
+          <p>
+            OpenStack uses a modular architecture with every service responsible for an individual cloud feature. This includes identity management, an image catalogue as well as all the compute, network and storage functions that are typically required to provision and decommission virtual machines.
+          </p>
+          <p>
+            The services are further broken down into processes based on a microservices architecture. Each process implements an atomic cloud function, such as handling user requests, proxying database queries or invoking various types of backend technologies used by OpenStack underneath.
+          </p>
+          <p>
+            In order to minimise its footprint, Canonical OpenStack ships with only the key features enabled by default. Additional features, such as CaaS or a Load Balancer can be enabled post-deployment with a single terminal command.
+          </p>
+        </div>
+        <hr class="is-muted" />
+        <a class="p-button--positive" href="/openstack/docs">Access supported features</a>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr />
+      <div class="col">
+        <div class="p-section--shallow">
+          <h2>OpenStack components</h2>
+        </div>
+      </div>
+      <div class="col">
+        <ul class="p-list--divided">
+          <li class="p-list__item is-ticked">Services expose APIs and handle user requests</li>
+          <li class="p-list__item is-ticked">Processes create a service and implement atomic cloud functions</li>
+          <li class="p-list__item is-ticked">Dashboard provides a web-based user interface to services</li>
+          <li class="p-list__item is-ticked">Client provides a command-line user interface to services</li>
+          <li class="p-list__item is-ticked">Databases store various types of records created by services</li>
+          <li class="p-list__item is-ticked">Queues facilitate communication between processes within a service</li>
+        </ul>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-9 col-start-large-4">
+        <hr class="is-muted u-hide--small" />
+        <div class="p-image-container u-hide--small">
```

Comment:
We should hide this since it isn't legible on medium screens 
```suggestion
        <div class="p-image-container u-hide--small u-hide--medium">
```